test-monitor/nitestmonitor.yml

```diff
@@ -0,0 +1,1277 @@
+      title: ResultsQueryResponse
+      type: object
+      required:
+        - results
+        - totalCount
+      properties:
+        results:
+          description: An array of test results
+          type: array
+          items:
+            $ref: '#/definitions/TestResultResponseObject'
+        totalCount:
+          description: The total count of matching test results
+          type: integer
+  StepsSuccessResponse:
+    description: Test Steps Success Response
+    schema:
+      description: Test Steps Success Response
+      title: StepsSuccessResponse
+      type: object
+      required:
+        - steps
+      properties:
+        steps:
+          description: An array of test steps
+          type: array
+          items:
+            $ref: '#/definitions/TestStepResponseObject'
+  StepsPartialSuccessResponse:
+    description: Test Steps Partial Success Response
+    schema:
+      description: Test Steps Partial Success Response
+      title: StepsPartialSuccessResponse
+      type: object
+      required:
+        - steps
+        - failed
+        - error
+      properties:
+        steps:
+          description: An array of test steps
+          type: array
+          items:
+            $ref: '#/definitions/TestStepResponseObject'
+        failed:
+          description: An array of test step requests that failed
+          type: array
+          items:
+            $ref: '#/definitions/TestStepRequestObject'
+        error:
+          $ref: '#/definitions/Error'
+  StepsQueryResponse:
+    description: Test Steps Query Response
+    schema:
+      description: Test Steps Query Response
+      title: StepsQueryResponse
+      type: object
+      required:
+        - steps
+        - totalCount
+      properties:
+        steps:
+          description: An array of test steps
+          type: array
+          items:
+            $ref: '#/definitions/TestStepResponseObject'
+        totalCount:
+          description: The total count of matching test steps
+          type: integer
+  StepsDeletePartialSuccessResponse:
+    description: Delete Test Steps Partial Success Response
+    schema:
+      description: Delete Test Steps Partial Success Response
+      title: StepsDeletePartialSuccessResponse
+      type: object
+      properties:
+        steps:
+          description: An array of test step id result id paris that were deleted
+          type: array
+          items:
+            $ref: '#/definitions/StepIdResultIdPair'
+        failed:
+          description: An array of test step id result id paris that failed to delete
+          type: array
+          items:
+            $ref: '#/definitions/StepIdResultIdPair'
+        error:
+          $ref: '#/definitions/Error'
+paths:
+  /:
+    get:
+      operationId: RootEndpoint
+      x-ni-request-all-privileges: true
+      summary: API information
+      description: Returns information about API versions and available operations.
+      tags: [versioning]
```

Comment:
Do you think we should try to be consistent with respect to how we order these properties across documents? In the message document I define tags, summary, description, operationId, and then responses, in that order. It might not be a big deal if we're at least consistent within each individual document.